.circleci/config.yml

```diff
@@ -26,6 +26,13 @@ default config for macOS: &macos_defaults
   macos:
     xcode: '12.1.0'
 
+
+config for macOS (android): &macos_defaults_android
+  <<: *defaults
+  resource_class: 'medium'
+  macos:
+    xcode: '11.5.0'
```

Comment:
Why can't we use `macos_defaults` for Android as well?

Reply:
I had to keep it at this version, due to android emulator not working. Maybe with detox upgrade we can use the same version, but I'll have to check it